src/main/logutils.ts

```diff
@@ -77,8 +85,54 @@ const handleLogLine = (line: string) => {
         handleRaidStopLine(line);
     } else if (line.includes("ZONE_CHANGE")) {
         handleZoneChange();
-    }  
-}    
+    } else if (line.includes("COMBATANT_INFO")) {
+        handleCombatantInfoLine(line);
+    } else if (line.includes("SPELL_AURA_APPLIED")){
+        handleSpellAuraAppliedLine(line);
+    }
+}
+
+/**
+ * Handles the SPELL_AURA_APPLIED line from WoW log.
+ * @param line the SPELL_AURA_APPLIED line
+ */
+const handleSpellAuraAppliedLine = (line: string) => {
+    if (combatantInfoMap.size > 0)
+    {
+        let sourcePlayerId = line.split(',')[2];
+        let sourceGUID = line.split(',')[3];
+        let playerData = combatantInfoMap.get(sourcePlayerId)
+        if (playerData != undefined)
+        {
+            playerData.friendly = isFriendlyUnit(parseInt(sourceGUID));
+        }
+    }
+}
+
+/**
+ * Handles the COMBATANT_INFO line from WoW log.
+ * @param line the COMBATANT_INFO line
+ */
+const handleCombatantInfoLine = (line: string) => {
+    let combatantData: CombatantData;
+    combatantData = {
+        friendly: false,
+        teamId: "",
+    }
+    const playerKey = line.split(',')[1];
+    combatantData.teamId = line.split(',')[2];
+    combatantInfoMap.set(playerKey, combatantData);
```

Comment:
`combatantInfoMap` needs to be reset at some point between matches

Reply:
addressed on this commit: https://github.com/aza547/wow-recorder/pull/25/commits/f493c35b35fe44a5feb4c120180f86ed19596fe2

Reply:
addressed here: https://github.com/aza547/wow-recorder/pull/25/commits/f493c35b35fe44a5feb4c120180f86ed19596fe2

Reply:
Still broken when players afk or disconnect, end event won’t get seen. Better off putting the reset in start

Reply:
Pretty sure this will be buggy when they play bgs/raids as well -- iirc the combatant_info event fires for those as well, which will add players to the map that werent part of the arena

Reply:
@ericlytle @aza547 👀

Reply:
Good points @prsauer. I've already merged... so some actions below: 

The afk/disconnect stuff: currently I think way worse things will happen on a DC (we'll just record forever). I should raise an issue to address that. Actually sounds like a tricky one to deal with. 

BG support: something I'm hoping to add over the weekend. I'll come back to this then. 

Raids: I think we just don't want to hit this code in raids, the outcome is trivial as it's in the ENCOUNTER_END event. Probably should gate this behind some condition. 

Another thought I had is performance. We only really need to hit the handleSpellAuraAppliedLine() once per arena game. I think currently we'll hit it every time the SPELL event fires. More conditional logic to fix this I think is needed.

Reply:
It might make sense to write a single end() function and call it every time something happens that would end a session (zone change, game closing, _end events, etc)